Sources/TensorFlow/Layers/Convolutional.swift

```diff
@@ -60,8 +65,11 @@ public struct Conv1D<Scalar: TensorFlowFloatingPoint>: Layer {
     /// - Returns: The output `[batchCount, newWidth, outputChannels]`.
     @differentiable
     public func callAsFunction(_ input: Tensor<Scalar>) -> Tensor<Scalar> {
-        let conv = conv2D(input.expandingShape(at: 1), filter: filter.expandingShape(at: 0),
-                          strides: (1, 1, stride, 1), padding: padding)
+        let conv = conv2D(input.expandingShape(at: 1),
```

Comment:
```suggestion
        let conv = conv2D(
            input.expandingShape(at: 1),
```

Can you align the arguments in a new line, like in the suggested edit?